shared/src/models/patient.ts

```diff
@@ -138,6 +141,10 @@ export class Patient {
     @Min(0)
     public readonly timeSpeed: number = 1;
 
+    @IsString()
+    @MaxLength(65535)
+    public readonly userText: string;
```

Comment:
```suggestion
    /**
     * This can be any arbitrary string. It gives trainers the freedom to add additional functionalities that are not natively supported by this application (like an hospital ticket system)
     */
    @IsString()
    @MaxLength(65535)
    public readonly freeTextField: string;
```

Reply:
or `remarks`